templates/oned.conf.erb

```diff
@@ -112,32 +119,47 @@ FEDERATION = [
     MASTER_ONED = ""
 ]
 
+<% if scope.function_versioncmp([@one_version,'4.14']) >= 0 -%>
+#*******************************************************************************
+# Default showback cost
+#-------------------------------------------------------------------------------
+# The following attributes define the default cost for Virtual Machines that
+# don't have a CPU, MEMORY or DISK cost. This is used by the oneshowback
+# calculate method.
+#*******************************************************************************
+
+DEFAULT_COST = [
+    CPU_COST    = 0,
+    MEMORY_COST = 0,
+    DISK_COST   = 0
+]
+<% end -%>
+
 #*******************************************************************************
 # XML-RPC server configuration
 #-------------------------------------------------------------------------------
 #  These are configuration parameters for oned's xmlrpc-c server
 #
 #  MAX_CONN: Maximum number of simultaneous TCP connections the server
-#  will maintain - default 15
+#  will maintain
 #
```

Comment:
Why have the mentioned default values been removed?
I would keep them in the config file.